addons/killtracker/stringtable.xml

```diff
@@ -86,5 +90,13 @@
             <Japanese>同士討ち</Japanese>
             <Turkish>Dost Atışı</Turkish>
         </Key>
+        <Key ID="STR_ACE_KillTracker_TrackAI_DisplayName">
+            <English>Track kills of AIs</English>
```

Comment:
I feel this is too ambiguous - imo it can be interpreted as both player AI kills or AI kills.